.circleci/config.yml

```diff
@@ -43,6 +52,22 @@ jobs:
       - codecov/upload:
         file: "./artifacts/0/artifacts/unit-tests/compile_commands.json"
 
+  app-center-publish:
+    <<: *container_medium
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Copy Configurations
+          command: cp MiniApp-Secrets.xcconfig ../MiniApp-Secrets.xcconfig
+      - run:
+          name: Deploy to AppCenter
+          no_output_timeout: 20m
+          command: ./scripts/deploy-to-app-center.sh
+      - run:
+          name: Clear Cache
+          command: rm ../MiniApp-Secrets.xcconfig
```

Comment:
what happens if deploy fails? is this command run anyway?

Reply:
Right. I think we can skip this even if it is failed.